modules/040-node-manager/template_tests/module_test.go

```diff
@@ -507,6 +508,20 @@ internal:
 var _ = Describe("Module :: node-manager :: helm template ::", func() {
 	f := SetupHelmConfig(``)
 
+	BeforeSuite(func() {
+		err := os.Symlink("/deckhouse/ee/modules/030-cloud-provider-openstack/cloud-instance-manager", "/deckhouse/modules/040-node-manager/cloud-providers/openstack")
```

Comment:
can we define global constants with paths ?

Reply:
I've extracted CIMs' paths to a global constants in https://github.com/deckhouse/deckhouse/pull/3740/commits/155087fb679297a6a3a51171691e704fd03f7086.